src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -45,6 +72,10 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, location
     # Setting subscription id
     subscription_id = get_subscription_id(cmd.cli_ctx)
 
+    # Setting resource Id
+    resource_id = "/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Kubernetes/connectedClusters/{}".format(subscription_id, resource_group_name, cluster_name)
+    telemetry.add_extension_event('connectedk8s', {'Context.Default.AzureCLI.ResourceId': resource_id})
```

Comment:
Azure CLI telemetry never records parameter values before. Please confirm with ddgdprapprove@microsoft.com for this property. @yungezz to review.

Reply:
Let me know if we can still check-in this change before the review of this property. It would save us time to create another PR for this after the review.

Reply:
Removed 'resourceid' from telemetry